src/FilterList.js

```diff
@@ -173,11 +173,33 @@ let FilterList = _.flow(
               style={bdJoin(child)}
             />
           ) : (
-            <div key={child.path} className="filter-list-item">
+            <div
+              key={child.path}
+              className="filter-list-item"
+              style={{
+                willChange: 'box-shadow',
+                transition: `box-shadow .4s ease-${
+                  child.paused ? 'in' : 'out'
+                }`,
+                boxShadow:
+                  child !== _.last(_.get('children', node)) && !child.paused
+                    ? 'rgba(0, 0, 0, 0.1) 0 -16px 20px -20px inset'
+                    : 'rgba(0, 0, 0, 0.1) 0 0 20px -20px inset',
```

Comment:
This seems like the realm of grey-vest, not contexture-react (which is still packaged in this lib). Alternate themes might not want an animated box shadow. The first thing that comes to mind would be to animate on and off a class that the grey-vest style sheet can target, but we did intend to move to emotion + theme UI in grey vest and kill the css in v3. I'm open to suggestions.

Reply:
Moved to GreyVest theme